script/deploy/defaultCollateral/DefaultCollateral.s.sol

```diff
@@ -0,0 +1,28 @@
+// SPDX-License-Identifier: UNLICENSED
```

Comment:
The script should work the way it works in suzaku-deployments or suzaku-deployer, that is, it should take a json file. It should also expose a `.json` file in deployments with the relevant deployment vars.

You can check more wow it's done here: https://github.com/suzaku-network/suzaku-deployments/blob/main/suzaku-protocol/script/collateral/DefaultCollateral.s.sol

You can slightly modify how deployments is exposed, as I don't think network-id/date/ is necessarily the best way.